packages/react-data-grid/src/Header.js

```diff
@@ -102,29 +105,30 @@ class Header extends React.Component {
         minHeight: rowHeight
       };
 
-      headerRows.push(<HeaderRow
-        key={row.ref}
-        ref={(node) => { return row.rowType === HeaderRowType.FILTER ? this.filterRow = node : this.row = node; }}
-        rowType={row.rowType}
-        style={headerRowStyle}
-        onColumnResize={this.onColumnResize}
-        onColumnResizeEnd={this.onColumnResizeEnd}
-        width={columnMetrics.width}
-        height={row.height || this.props.height}
-        columns={columnMetrics.columns}
-        resizing={resizeColumn}
-        draggableHeaderCell={this.props.draggableHeaderCell}
-        filterable={row.filterable}
-        onFilterChange={row.onFilterChange}
-        onHeaderDrop={this.props.onHeaderDrop}
-        sortColumn={this.props.sortColumn}
-        sortDirection={this.props.sortDirection}
-        onSort={this.props.onSort}
-        onScroll={this.props.onScroll}
-        getValidFilterValues={this.props.getValidFilterValues}
-        />);
+      return (
+        <HeaderRow
+          key={row.rowType}
```

Comment:
Not sure why was `key` a function (`row.ref` defined in [`ReactDataGrid.js`](https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/ReactDataGrid.js#L561))